Foundation/DateIntervalFormatter.swift

```diff
@@ -133,8 +133,13 @@ open class DateIntervalFormatter: Formatter {
         var timeStyle: Int64 = 0
         var dateTemplate: Unmanaged<CFString>?
         var dateTemplateFromStyles: Unmanaged<CFString>?
+#if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
+        var modified: DarwinBoolean = false
+        var useTemplate: DarwinBoolean = false
+#else
         var modified: Bool = false
         var useTemplate: Bool = false
+#endif
         var locale: Unmanaged<CFLocale>?
```

Comment:
Would it be better to just use `_DarwinCompatibleBoolean` instead of spliting it as the following is defined in `NSSwiftRuntime.swift`

```
#if os(macOS) || os(iOS)
    internal typealias _DarwinCompatibleBoolean = DarwinBoolean
#else
    internal typealias _DarwinCompatibleBoolean = Bool
#endif
```

Reply:
Um, it would, but that requires being aware of `_DarwinCompatibleBoolean`.  I cant believe that I've been trying to figure this out for nearly a week :/